From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Natalie Vock <natalie.vock@gmx.de>
Date: Fri, 29 May 2026 17:15:05 +0200
Subject: [PATCH 10/17] STEAMOS: driconf: Enable VM explicit sync for Forza
 Horizon 5/6

To mitigate gameplay stutters.

(cherry picked from commit 70a35fb1fa62e617c25bdf064dbeec386bf9ce0d)
---
 src/amd/vulkan/00-radv-defaults.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/amd/vulkan/00-radv-defaults.conf b/src/amd/vulkan/00-radv-defaults.conf
index 9dbedb18076..1b5f9241d4e 100644
--- a/src/amd/vulkan/00-radv-defaults.conf
+++ b/src/amd/vulkan/00-radv-defaults.conf
@@ -235,6 +235,8 @@ Application bugs worked around in this file:
             <!-- Force waiting for VM MAP updates at allocation time to
                  workaround an use-before-alloc which causes GPU hangs. -->
             <option name="radv_wait_for_vm_map_updates" value="true" />
+            <!-- Use explicit sync to mitigate stutters in gameplay -->
+            <option name="radv_vm_explicit_sync" value="true" />
         </application>
 
         <application name="Forza Horizon 6" application_name_match="forzahorizon6.exe">
@@ -246,6 +248,8 @@ Application bugs worked around in this file:
                  TODO Remove this when VK_EXT_descriptor_heap is enabled by
                  default because it should not be necessary anymore. -->
             <option name="radv_force_64_byte_sampled_image" value="true" />
+            <!-- Use explicit sync to mitigate stutters in gameplay -->
+            <option name="radv_vm_explicit_sync" value="true" />
         </application>
 
         <application name="Far Cry 2" application_name_match="FarCry2.exe">
